ansible/group_vars/all.yaml

```diff
@@ -0,0 +1,26 @@
+ethereum_helm_chart_url: https://skylenet.github.io/ethereum-helm-charts
+
+# for k3s traefik is installed by default
+ingress_controller: traefik
+
+# Change accordingly -> xdai or sokol
+ethereum_network: xdai
```

Comment:
If we already have an evn file, can we reference the value of the `ethereum_network` from that file?

Reply:
I guess we can do that - but what is the benefit of that?